chipy_org/deploy/wsgi.py

```diff
@@ -1,4 +1,14 @@
-from django.core.wsgi import get_wsgi_application
-from dj_static import Cling
+"""
+WSGI config for chipy2 project.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.6/howto/deployment/wsgi/
+"""
 
-application = Cling(get_wsgi_application())
```

Comment:
Thanks for the clarification on that.  I'm adding this back.  I must have overwritten this file when I copied in the new Django project structure.